FizzBuzz/Program.cs

```diff
@@ -0,0 +1,57 @@
+﻿/*FizzBuzz
+ * A program that writes numbers from 1 to 100. Any number that is a multiple of three is replaced
+ * by the word 'Fizz', any number that is a multiple of five is replaced by the word 'Buzz', and any
+ * number that is a multiple of both three and five is replaced by the word 'FizzBuzz'.
+ */
+
+using System;
+
+namespace FizzBuzz
+{
+    class Program
+    {
+            static void Main(string[] args)
+            {
+                double ii;
+
+
+                for (ii = 1; ii <= 100; ii += 1)
+                {
+                    double div1 = (ii / 3);
+                    double div2 = (ii / 5);
+
+                    //Console.WriteLine(div1);
+                    //Console.WriteLine(div2);
+
+                    if (((div1 % 1) == 0) && ((div2 % 1) == 0))
```

Comment:
You could actually skip the dividing entirely and just use modulo 3 and modulo 5 here instead of mod 1